### DISCUSSION TITLE
Save Stellarium Settings for a Push To Scope?

### DISCUSSION THREAD

**Discussion by TonyMan13** [Stellarium/stellarium#3329] [Q&A]
A club member hit me with this tonight. We have a scope on an equatorial fork mount attached to a concrete block in one of our observatories. It's clock driven so it tracks. We recently attached a circuit that allows us to use Stellarium and push the scope to new objects watching the setting circle counts. That works fine. Here is the question that came up: Since the scope in mounted permanently in one location and can't be moved from there, both ASCOM and Stellarium have the longitude and latitude, and the PC provides date and time of day, why do we have to do a two star alignment every time we use it? Is there some way all that info can be saved to a file that can either be autoloaded when Stellarium starts, or loaded by the user? The more we talked about it, the more logical it sounded, we just didn't see such an option in Stellarium.

Any ideas appreciated.

**Comment by gzotti** [comment]
In a different forum somebody else gave the answer to you. Stellarium has no PARK function. Stellarium assumes that you have set up your telescope correctly with the means provided by the telescope's drive before connecting and then sends SLEW commands and reports back position.

**Comment by TonyMan13** [comment]
Thanks. Since we have a basically dumb mount where we just use software to watch the encoder clicks, I understand that parking is not an option.
